server/documents/modules/rating.html.eco

```diff
@@ -83,6 +83,29 @@ themes      : ['Default']
       <br><br>
     </div>
 
+    <div class="example">
+      <h4 class="ui header">Partial Ratings </h4>
```

Comment:
Please add `<div class="ui black label">New in 2.8.0</div>` before the `</h4>`  when your PR in the main repo gets merged

Reply:
@zombietfk bump